File: cmd/debtdrone/testdata/dirty_code/complex.ts
function complexFunction(x: number): void {
    if (x > 0) {
        for (let i = 0; i < x; i++) {
            if (i % 2 === 0) {
                if (i % 3 === 0) {
                    for (let j = 0; j < i; j++) {
                        if (j > 5) {
                            console.log("Deeply nested");
                        }
                    }
                }
            }
        }
    } else {
        if (x < -5) {
            console.log("Negative");
        }
    }
}

File: cmd/debtdrone/testdata/dirty_code/complex.rs
fn complex_function(x: i32) {
    if x > 0 {
        for i in 0..x {
            if i % 2 == 0 {
                if i % 3 == 0 {
                    for j in 0..i {
                        if j > 5 {
                            println!("Deeply nested");
                        }
                    }
                }
            }
        }
    } else {
        if x < -5 {
            println!("Negative");
        }
    }
}
